poleno/utils/admin.py

```diff
@@ -63,3 +65,17 @@ def get_actions(self, request):
         actions = super(NoBulkDeleteAdminMixin, self).get_actions(request)
         actions.pop(u'delete_selected', None)
         return actions
+
+class DownloadAdminMixin(admin.ModelAdmin):
```

Comment:
Nebolo by lepsie ten mixin definovat v `poleno/attachments/admin.py`? Kedze to nie je uplne vseobecna utilita, ale attachment-specific utilita. Ved aj v nom pouzivas `download` z `poleno.attachments.views`. Vseobecne utility by nemali includovat ine apky.

Reply:
:ok: